things/api/things/http/endpoint.go

```diff
@@ -476,6 +476,21 @@ func createConnectionsEndpoint(svc things.Service) endpoint.Endpoint {
 	}
 }
 
+func createDisconnectionsEndpoint(svc things.Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		cr := request.(createConnectionsReq)
```

Comment:
Can you use `req` instead? For sake of consistency in other endpoints. It's scoped as  `req` , which makes more sense for readability also.